### DISCUSSION TITLE
Configure Nominatim image Docker with remote PostgreSql

### DISCUSSION THREAD

**Discussion by lucascognamiglio1** [mediagis/nominatim-docker#300] [Q&A]
Hello
Is it possible to configure the image Docker to use nominatim with a remote instance of PostgreSql?

**Comment by leonardehrenfried** [comment]
I have never tried it but people have mixed results: https://github.com/mediagis/nominatim-docker/issues/245

Can you try it and let us know?

**Reply by mtmail** [reply]
Nominatim 4.0's new ICU tokenizer no longer needs the legacy`nominatim.so` file so I think it's easier now. In the past one had to copy the file to the Postgresql Server's file system which required extra steps and was often impossible with managed cloud databases. You have to enable the new tokenizer https://nominatim.org/release-docs/4.0.1/customize/Tokenizers/#icu-tokenizer
